MyLibrary/MyLibraryApp.Application/Models/Books/DeleteBookRequestModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using MyLibraryApp.DataAccess.Entities;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.Application.Models.Books
+{
+    public class DeleteBookRequestModel
+    {
+        [Required]
+        public int Id { get; set; }
+
+        [Required]
+        public string Title { get; set; }
```

Comment:
I don't think all these details are needed when you want to remove a book. Maximum the id in combination with the book tile but even that could be too much. Only the id should be enough.